sdk/src/beta9/abstractions/endpoint.py

```diff
@@ -439,18 +440,25 @@ async def _heartbeat():
 
                         internal_asgi_app.input_queue.put(data)
 
+                        async def _handle_output(output):
```

Comment:
Were you able to test this with all these cases?

Reply:
Yep. str and dict/list works for both returnable functions and generators